test/prototype_transforms_kernel_infos.py

```diff
@@ -631,7 +658,9 @@ def reference_inputs_convert_format_bounding_box():
 
 
 def sample_inputs_convert_color_space_image_tensor():
-    color_spaces = list(set(features.ColorSpace) - {features.ColorSpace.OTHER})
+    color_spaces = sorted(
+        set(features.ColorSpace) - {features.ColorSpace.OTHER}, key=lambda color_space: color_space.value
+    )
```

Comment:
Drive-by that I found while debugging. Without this, individual failing tests are not reproducible because the order is not stable.